tests/async/test_assertions.py

```diff
@@ -161,7 +161,7 @@ async def test_assertions_locator_to_contain_class(page: Page, server: Server) -
 
     assert excinfo.match("Locator expected to contain class 'does-not-exist'")
     assert excinfo.match("Actual value: foo bar baz")
-    assert excinfo.match("LocatorAssertions.to_contain_class with timeout 100ms")
+    assert excinfo.match('Expect "to.contain.class" with timeout 100ms')
```

Comment:
Looks like a bug? I think it should include to_contain_text or even better `expect(locator).to_contain_class`.